pom.xml

```diff
@@ -11,15 +11,15 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>video-js</artifactId>
-    <version>4.11.5-SNAPSHOT</version>
+    <version>5.2.1</version>
     <name>Video.js</name>
     <description>WebJar for Video.js</description>
     <url>http://webjars.org</url>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <upstream.version>4.11.4</upstream.version>
-        <upstream.url>http://www.videojs.com/downloads</upstream.url>
+        <upstream.version>5.2.1</upstream.version>
+        <upstream.url>https://github.com/videojs/video.js/archive/v5.2.1/video-js-5.2.1.zip</upstream.url>
```

Comment:
When I modified the upstream url it generates an issue, As you can see on the link bellow.
https://ncry.pt/p/2lBn#9UlAvmNRHKt6zAAzc7OnNjileuJwr5cAPn-Ydjg6oHI

But if I change it with this one it is working
https://github.com/videojs/video.js/archive/v5.2.1/video-js-5.2.1.zip